### DISCUSSION TITLE
Incompatible with widely used plugins and little to no customization features

### DISCUSSION THREAD

**Discussion by skinnerjoe** [woocommerce/woocommerce#44733] [Checkout Flow]
When will this support "Braintree by Payment Plugins" and "Woocommerce Smart Coupons"?
The official Braintree by Woocommerce plugin doesn't have options such as Google Pay and Venmo so switching to that plugin is not a viable solution.
These new blocks have been available for some time now, yet they do not support some of the most widely used plugins.

**Comment by senadir** [comment]
Hey! I understand this discussion is related to the one here https://github.com/woocommerce/woocommerce/discussions/45259

So I'm closing this one and continuing there.
As for your request, we do not control integrations, it should be done by the Payment Plugins team, which you're probalby aware of by now.
